EmbedCases.cmake

```diff
@@ -50,4 +50,9 @@ endmacro (opm_pack_stone)
 opm_pack_stone (upscale_relperm_benchmark)
 opm_pack_case (upscale_relperm_benchmark benchmark20)
 
-add_custom_target(benchmarks DEPENDS upscale_relperm_benchmark)
+if(INSTALL_BENCHMARKS)
+	add_custom_target(benchmarks ALL DEPENDS upscale_relperm_benchmark)
```

Comment:
benchmarks is a custom target, which all the specific benchmarks depends on.
i then mark the *specific target* as not-excluded-from-all. reason for this is to quell a cmake warning - 'trying to install a target that is excluded from all is undefined'.